<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mxrepomanager</class>
 <widget class="QDialog" name="mxrepomanager">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MX Repo Manager</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select the APT repository that you want to use:</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMX">
      <attribute name="title">
       <string>Default MX repo</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="autoScroll">
          <bool>false</bool>
         </property>
         <property name="autoScrollMargin">
          <number>16</number>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="movement">
          <enum>QListView::Static</enum>
         </property>
         <property name="flow">
          <enum>QListView::TopToBottom</enum>
         </property>
         <property name="isWrapping" stdset="0">
          <bool>false</bool>
         </property>
         <property name="resizeMode">
          <enum>QListView::Adjust</enum>
         </property>
         <property name="layoutMode">
          <enum>QListView::SinglePass</enum>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="viewMode">
          <enum>QListView::ListMode</enum>
         </property>
         <property name="selectionRectVisible">
          <bool>false</bool>
         </property>
         <property name="currentRow">
          <number>-1</number>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDebian">
      <attribute name="title">
       <string>Debian repos</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushFastestDebian">
         <property name="text">
          <string>Select fastest Debian repos for me</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QTreeWidget" name="treeWidgetDeb">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAllRepos">
      <attribute name="title">
       <string>Individual sources</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="buttonBar" columnstretch="0,0,0,0,0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="0" column="1">
      <widget class="QPushButton" name="buttonHelp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Display help </string>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../../../usr/share/mx-repo-manager/icons/gnome-help.png</normaloff>../../../../../../../../usr/share/mx-repo-manager/icons/gnome-help.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+H</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="buttonAbout">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>About this application</string>
       </property>
       <property name="text">
        <string>About...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../../../usr/share/mx-repo-manager/icons/accessories-text-editor.png</normaloff>../../../../../../../../usr/share/mx-repo-manager/icons/accessories-text-editor.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+B</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="buttonCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Quit application</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../../../usr/share/mx-repo-manager/icons/gtk-cancel.png</normaloff>../../../../../../../../usr/share/mx-repo-manager/icons/gtk-cancel.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+N</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="labelMX">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../../../../../usr/share/mx-repo-manager/icons/mxfcelogo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="buttonOK">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../../../usr/share/mx-repo-manager/icons/dialog-ok.png</normaloff>../../../../../../../usr/share/mx-repo-manager/icons/dialog-ok.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>buttonAbout</tabstop>
  <tabstop>buttonHelp</tabstop>
  <tabstop>buttonOK</tabstop>
  <tabstop>buttonCancel</tabstop>
  <tabstop>treeWidgetDeb</tabstop>
  <tabstop>treeWidget</tabstop>
  <tabstop>pushFastestDebian</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>pressed()</signal>
   <receiver>mxrepomanager</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>766</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
